### PR TITLE
add ptr correspondence types to portal batch change docs

### PR DESCRIPTION
**Pull request by britneywright**
doc update in support of #511

### CODE REVIEW

File: modules/docs/src/main/tut/portal/batch-changes.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Batch change is an alternative to submitting individual RecordSet changes and pr
 To update an existing record, you must delete the record first and add all expected records within the same batch; a delete and add of the same recordset within a batch change request will be processed as an update.
 
 Batch changes are also susceptible to the following restrictions:
-* Current supported record types for batch change are: **A**, **AAAA**, **CNAME**, **PTR**, **TXT**, and **MX**.
+* Current supported record types for batch change are: **A**, **A+PTR**, **AAAA**, **AAAA+PTR**, **CNAME**, **PTR**, **TXT**, and **MX**.
```
Comment (rebstar6) [inline modules/docs/src/main/tut/portal/batch-changes.md]: actually, split out to new sentence, something like:

Current supported record types for batch change are: **A**, **AAAA**, **CNAME**, **PTR**, **TXT**, and **MX**.
In addition, there are  **A+PTR** and **AAAA+PTR** types; note that behind the scenes, these will be processed as separate A (or AAAA) and PTR changes, and will appear in the final change as separate items.

File: modules/docs/src/main/tut/portal/batch-changes.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Batch change is an alternative to submitting individual RecordSet changes and pr
 To update an existing record, you must delete the record first and add all expected records within the same batch; a delete and add of the same recordset within a batch change request will be processed as an update.
 
 Batch changes are also susceptible to the following restrictions:
-* Current supported record types for batch change are: **A**, **AAAA**, **CNAME**, **PTR**, **TXT**, and **MX**.
+* Current supported record types for batch change are: **A**, **AAAA**, **CNAME**, **PTR**, **TXT**, and **MX**. Additionally, there are **A+PTR** and **AAAA+PTR** types that, behind the scenes, will be processed as separate A (or AAAA) and PTR changes, and will appear in the completed batch change summary as separate items.
```
Comment (mitruly) [inline modules/docs/src/main/tut/portal/batch-changes.md]: I feel like this seems a bit verbose. Can we include `A+PTR` and `AAAA+PTR` with the other supported record types and link them to a footnote that talks about how they will be processed?

Comment (britneywright) [reply modules/docs/src/main/tut/portal/batch-changes.md]: we don't do footnotes and I don't think this is the place to start. I don't have strong opinions on how much or how little we explain it though.

Comment (mitruly) [reply modules/docs/src/main/tut/portal/batch-changes.md]: If that's the case, then how about something like:
```
Additionally, there are **A+PTR** and **AAAA+PTR** types that will be processed as a separate A (or AAAA) and PTR change, and will also appear separately in the completed batch change summary.
```

**Pull request merged**